File: Swap First and Last Element/SwapFirstAndLastElement.java
public class SwapFirstAndLastElement {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int n = sc.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + ": ");
            a[i] = sc.nextInt();
        }
        int temp = a[0];
        a[0] = a[n - 1];
        a[n - 1] = temp;
        System.out.println("Array after swapping first and last element:");
        for (int i = 0; i < n; i++) {
            System.out.println(a[i]);
        }
    }
}

File: Swap First and Last Element/swap_first_and_last_element.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    int temp = a[0];
    a[0] = a[n - 1];
    a[n - 1] = temp;
    printf("Array after swapping first and last element:\n");
    for (int i = 0; i < n; i++) {
        printf("%d\n", a[i]);
    }
    return 0;
}
